RecoTracker/IterativeTracking/python/MixedTripletStep_cff.py

```diff
@@ -161,6 +169,18 @@
 )
 mixedTripletStepHitTripletsB = mixedTripletStepHitTripletsA.clone(doublets = "mixedTripletStepHitDoubletsB")
 mixedTripletStepSeedsB = mixedTripletStepSeedsA.clone(seedingHitSets = "mixedTripletStepHitTripletsB")
+#fastsim
+from FastSimulation.Tracking.SeedingMigration import _hitSetProducerToFactoryPSet
+_fastSim_mixedTripletStepSeedsA.seedFinderSelector.pixelTripletGeneratorFactory = _hitSetProducerToFactoryPSet(mixedTripletStepHitTripletsA)
+fastSim.toReplaceWith(mixedTripletStepSeedsA,_fastSim_mixedTripletStepSeedsA)
```

Comment:
Ideally I'd like that these customizations would be placed after the other definitions of `mixedTripletStepSeedsA`. I presume the fact that `mixedTripletStepSeedsB` is cloned from `mixedTripletStepSeedsA` with a parameter not in FastSim configuration motivated the choice here.

How about something like this (following my suggestion of a similar ordering/parameter inconsistency with the XeXe tracking in #20749):
```python
_mixedTripletStepSeedsCommon = _seedCreatorFromRegionConsecutiveHitsTripletOnlyEDProducer.clone(...)
trackingLowPU.toModify(_mixedTripletStepSeedsCommon, ...)
mixedTripletStepSeedsA = _mixedTripletStepSeedsCommon.clone()
...
fastSim.toReplaceWith(mixedTripletStepSeedsA, _fastSim_mixedTripletStepSeedsA)

mixedTripletStepSeedsB = _mixedTripletStepSeedsCommon.clone(seedingHitSets = "mixedTripletStepHitTripletsB")
...
fastSim.toReplaceWith(mixedTripletStepSeedsB, _fastSim_mixedTripletStepSeedsB)
```

Reply:
I merged Chris' branch into my branch. I am testing my changes before I commit them on  my branch.

Reply:
@makortel This pull request came out of what we discovered in  #20879. I've kept the two seperate for now since they both can be a bit controversial. Plus, it is easier to test them independently to be sure they both do what they are intended.

Reply:
Ok, understood. But what about my comment for this piece of configuration?